examples/benchmarks/blas.cpp

```diff
@@ -25,11 +26,18 @@ int main(int argc, char** argv) {
     try {
         int device = argc > 1 ? atoi(argv[1]) : 0;
         setDevice(device);
+
+        const std::string dtype(argc > 2 ? argv[2] : "f32");
+        const af_dtype dt = (dtype == "f16" ? f16 : f32);
+
+        if (dt == f16)
+          printf("Device %d isHalfAvailable ? %s\n", device, isHalfAvailable(device) ? "yes" : "no");
+
         info();
 
-        printf("Benchmark N-by-N matrix multiply\n");
+        printf("Benchmark N-by-N matrix multiply at %d \n", dtype);
```

Comment:
`dtype` is a string. I think you mean `dt` here.

Reply:
oops I meant %s. fixing.

Reply:
replaced %d by %s